tslint.json

```diff
@@ -36,7 +36,7 @@
 			"match-default-export-name": true,
 			"mocha-avoid-only": true,
 			"mocha-no-side-effect-code": true,
-			"no-any": true,
+			"no-any": false,
```

Comment:
just disable the no-any where you need it, in the line above or a block, but not disable all no-any checks